<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountItem</class>
 <widget class="QWidget" name="AccountItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="iterativeCheckBox">
     <property name="text">
      <string>iterative</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="startDateTimeEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;start time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="endDateTimeEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;end time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="amountSpinBox">
     <property name="suffix">
      <string> hours</string>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="startTimeEdit"/>
   </item>
   <item>
    <widget class="QTimeEdit" name="endTimeEdit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item>
    <widget class="QComboBox" name="typeComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
